.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ fail_fast: false
 
 repos:
   - repo: https://github.com/psf/black
-    rev: 22.1.0
+    rev: 22.6.0
```

Comment:
The current version has a dependency clash with `click`; anyway, this doesn't affect anything in the production code.